draft-nygren-httpbis-httpssvc.md

```diff
@@ -455,9 +455,31 @@ received over HTTPS to those received via DNS.
 
 # Client behaviors
 
-TODO: Write more details on the recommended resolution algorithm.
-This section does NOT yet cover the SvcRecordType=0 case well yet.
+## Client resolution {#resolution}
 
+When attempting to resolve a name HOST, clients should follow in-order:
+
+1. Issue parallel AAAA, A, and HTTPSSVC queries for the name HOST.
+   The answers for these may or may not include CNAME pointers
+   before reaching one or more of these records.
+   
+2. If an HTTPSSVC record of SvcRecordType "0" is returned for HOST,
+   clients should loop back to step 1 replacing HOST with SvcDomainName.
```

Comment:
Any loop detection logic?  Could be as simple as "subject to loop detection heuristics".